File: backend/src/main/java/ProgramaClip/ClipMoneyAPI/entidad/Factura.java
package ProgramaClip.ClipMoneyAPI.entidad;

import java.sql.Date;
import java.sql.Time;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.ManyToOne;

import com.sun.istack.NotNull;

import lombok.Data;

@Entity
@Data
public class Factura {
	@Id
	@GeneratedValue
	private Long id;
	@NotNull
	private Long nroFactura;
	@NotNull
	private Float monto;
	@NotNull
	private Date fechaPrimerVencimiento;
	private Date fechaSegundoVencimiento;
	@NotNull
	private Float montoPrimerVencimiento;
	private Float montoSegundoVencimiento;
	private Date fechaPago;
	private Time horaPago;
	@ManyToOne
	@NotNull
	private Servicio servicio;
	@ManyToOne
	private Cuenta cuentaPago;
	@ManyToOne
	@NotNull
	private EstadoFactura estado;
}

File: Entidades/Factura.cs
namespace proyecto_clip_money_e_proyecto_clip.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    public partial class Factura
    {
        public long Id { get; set; }

        public long NroFactura { get; set; }

        public double Monto { get; set; }

        [Column(TypeName = "date")]
        public DateTime FechaPrimerVencimiento { get; set; }

        [Column(TypeName = "date")]
        public DateTime? FechaSegundoVencimiento { get; set; }

        public double MontoPrimerVencimiento { get; set; }

        public double? MontoSegundoVencimiento { get; set; }

        public DateTime? FechaHoraPago { get; set; }

        public long ServicioId { get; set; }

        public long? CuentaPagoNroCuenta { get; set; }

        public int EstadoFacturaId { get; set; }

        public long UsuarioId { get; set; }

        public virtual Cuenta CuentaPago { get; set; }

        public virtual EstadoFactura EstadoFactura { get; set; }

        public virtual Servicio Servicio { get; set; }

        public virtual Usuario  Usuario { get; set; }
    }
}
